images/vm/alpine/alpine.yml.in

```diff
@@ -9,21 +9,28 @@ init:
 onboot:
   - name: sysctl
     image: linuxkit/sysctl:v0.7
+  - name: busybox-checkdev
+    image: busybox:musl
+    command: ["sh", "-c", "mkdir /tmp/mnt && ls /dev/sr0 &&ln -s /dev/sr0 /tmp/metadatadev||ln -s /dev/vdb /tmp/metadatadev"]
+    capabilities:
+      - CAP_SYS_ADMIN
+    binds:
+      - /tmp:/tmp
+      - /dev:/dev
   - name: mounts
     image: linuxkit/mount:a718496fa5ea2a7a9f7552eac64c7f3699fb6a86
-    command: ["/usr/bin/mountie", "/dev/sr0", "/mnt"]
+    command: ["/usr/bin/mountie", "/tmp/metadatadev", "/tmp/mnt"]
```

Comment:
All of this seems to be done as to work around linuxkit view of where metadata needs to be mounted. While this is OK for now -- I think we need to invest in fixing/improving linuxkit itself. So @sadov I suggest you prepare those linuxkit PRs and @deitch can help usptreaminng them.

In fact, if you have any questions on how to fix it in linuxkit -- you can ping either myself or @deitch too